scipy/stats/tests/test_binned_statistic.py

```diff
@@ -1,7 +1,7 @@
 from __future__ import division, print_function, absolute_import
 
 import numpy as np
-from numpy.testing import assert_allclose
+from numpy.testing import assert_allclose, assert_raises
```

Comment:
I think you have to use `pytest.raises` instead of `assert_raises` since it uses `nose` internally.

Reply:
> I think you have to use `pytest.raises` instead of `assert_raises` since it uses `nose` internally.

this is weird b/c it passes locally, I can change to:
`from pytest import raises as assert_raises` as a new line (b/c we still need `assert_allclose`), for my own edification I'd like to understand why this passes locally. Is this a difference in configuration of travis vs my local build env? Or is this some sort of (currently occuring migration in `numpy testing` from `nose` to `pytest`

Reply:
You probably have `nose` installed locally so `numpy.testing` can import it without issue.  Travis intentionally doesn't have `nose` installed though: see gh-10409.

Reply:
Cool, yes I do have local install. I had missed gh-10409 when that went in last month :) Thanks @peterbell10 !

Reply:
Went from 3 failing builds to 2, so I guess that is progress :). ONe of the fails looks like the sphinx docs are failing. I'll look into that more tomorrow (or today now) in the evening.

Reply:
everything is green now and I address your comments @peterbell10 please let me know if you would like other changes.